forced_photometry/multiband_photometry.ipynb

```diff
@@ -503,19 +458,75 @@
     "prf_nuv = prf_nuv[(prf_cen - int(ngalex_pix / 2) - 1) : (prf_cen + int(ngalex_pix / 2)),\n",
     "                  (prf_cen - int(ngalex_pix / 2) - 1) : (prf_cen + int(ngalex_pix / 2))]\n",
     "prf_fuv = prf_fuv[(prf_cen - int(ngalex_pix / 2) - 1) : (prf_cen + int(ngalex_pix / 2)),\n",
-    "                  (prf_cen - int(ngalex_pix / 2) - 1) : (prf_cen + int(ngalex_pix / 2))]\n",
+    "                  (prf_cen - int(ngalex_pix / 2) - 1) : (prf_cen + int(ngalex_pix / 2))]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# prfs and other configs for each channel\n",
     "\n",
+    "# container for configs of a single band\n",
+    "class BandConfigs(NamedTuple):\n",
```

Comment:
I'm not sure that `BandConfigs` is the right name for this class (and the related `band_configs` variables below) -- specifically the "configs" part. This is something the user may interact with, so I'd like it to be well-named such that its meaning/contents is clear to the user. Seeking input @jkrick .

Reply:
Documenting a synchronous conversation: decided `BandParams` would be better.